presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8088,4 +8088,22 @@ private List<MaterializedRow> getNativeWorkerSessionProperties(List<Materialized
                 .filter(row -> Pattern.matches(sessionPropertyName, row.getFields().get(4).toString()))
                 .collect(toList());
     }
+
+    /**
+     By default, casts only when storageFormat is DWRF.
+     */
+    protected String getOrderDateExpression(String storageFormat)
+    {
+        // DWRF does not support date type.
+        return storageFormat.equals("DWRF") ? "cast(o.orderdate as DATE)" : "o.orderdate";
+    }
+
+    /**
+     By default, casts only when storageFormat is DWRF.
+     */
+    protected String getShipDateExpression(String storageFormat)
+    {
+        // DWRF does not support date type.
+        return storageFormat.equals("DWRF") ? "cast(l.shipdate as DATE)" : "l.shipdate";
+    }
```

Comment:
```suggestion

   /**
    * Returns a date expression, casting to DATE if storageFormat is DWRF.
    */
    protected String getDateExpression(String storageFormat, String columnExpression) 
    {
        // DWRF does not support date type.
        return storageFormat.equals("DWRF") ? "cast(" + columnExpression + " as DATE)" : columnExpression;
    }
```

nit: good to refactor